Publish fails during manifest generation when too many images in the repo

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
When there are too many docker images in the docker repo, the CI fails to publish the images. For example, see [this master build](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/master/1861/pipeline):
```
time="2020-04-06T19:25:17Z" level=error msg="Error trying v2 registry: http: unexpected EOF reading trailer"
time="2020-04-06T19:25:17Z" level=fatal msg="Inspect of image \"rook/cockroachdb-arm64:v1.3.0.62.g099ed30\" failed with error: http: unexpected EOF reading trailer"
Makefile:183: recipe for target 'publish.manifest.image.cockroachdb' failed
make: *** [publish.manifest.image.cockroachdb] Error 1
```

This issue is impacting Rook's three oldest operators which have the most images pushed: Ceph, CockroachDB, and NFS. 

As a workaround, old unused master tags have been deleted from the dockerhub repo.

Looks like there is a fix in the manifest tool for this.
https://github.com/estesp/manifest-tool/pull/90

**Expected behavior:**
Publishing images should not be limited by the number of images in the repo.

--- a/build/release/Makefile
+++ b/build/release/Makefile
@@ -74,7 +74,7 @@ S3_SYNC_DEL := aws s3 sync --only-show-errors --delete
 # ====================================================================================
 # tools
 
-MANIFEST_TOOL_VERSION=v1.0.0
+MANIFEST_TOOL_VERSION=v1.0.2
 MANIFEST_TOOL := $(TOOLS_HOST_DIR)/manifest-tool-$(MANIFEST_TOOL_VERSION)
 
 $(MANIFEST_TOOL):

